### PR TITLE
Work aboutme

### PR DESCRIPTION
**Pull request by IamYunusAli**
In this pull  I used Flexbox and Grid, I have accomplished the following;
 Added Work section 
 Added about me 

Grid for work layout and flex for the items in it

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -97,3 +112,178 @@ li:hover {
...
+/* work section */
+.work {
+  background-color: #e5e5e5;
+  margin: 100px 0;
+  align-items: center;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.container {
```
Comment (tufoinnkuo10) [inline style.css:138]: - [ ] great job with the project .please make sure the different content of projects are displayed with the flexbox

File: style.css [hunk 2]
```diff
@@ -97,3 +112,178 @@ li:hover {
 li:active {
   color: #2230d2;
 }
+
+/* work section */
+.work {
+  background-color: #e5e5e5;
```
Comment (tufoinnkuo10) [inline style.css]: - [ ] please use the correct work section background color

File: index.html [hunk 3]
```diff
@@ -19,28 +19,167 @@ <h4>Yunus Ali</h4>
...
+                            <li><a href="#"><i class="bi bi-medium"></i></a></li>
+                            <li><a href="#"><i class="bi bi-github"></i></a></li>
+                            <li><a href="#"><i class="bi bi-slack"></i></a></li>
+                    </ul>
+                </section>
+            </article>
+            </section>
+
+            <section class="work">
+                    <div class="container">
```
Comment (tufoinnkuo10) [inline index.html:52]: - [ ] kindly fix the padding/gap between your headline section and work section.
your design
Screenshot (66)_LI
figma design
Screenshot (67)_LI

File: index.html [hunk 4]
```diff
@@ -19,28 +19,167 @@ <h4>Yunus Ali</h4>
...
+                        I’m a software developer!
+                        I can help you build a product ,
+                        feature or website Look through
+                        some of my work and experience!
+                        If you like what you see and have
+                        a project you need coded, don’t hestiate
+                        to contact me.
+                    </p>
+                    <section class="connect">
+                        <h4>Lets connect</h4>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please use thesame figma font size here

File: index.html [hunk 5]
```diff
@@ -19,28 +19,167 @@ <h4>Yunus Ali</h4>
...
+                          </ul>
+                        <br>
+                        <button class='btn' type="button">See projects</button>
+                      </div>
+
+            </section>
+            <section class="aboutme">
+
+                <article class="aboutme-article">
+                    <h1>About Myself</h1>
```
Comment (tufoinnkuo10) [inline index.html]: - [ optional] you can break the `about myself` using `<be>` so it looks like the figma version

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @IamYunusAli   

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Good commit messages
✅ Good READMe template
✅ Great pull request summary

Required Changes ♻️
Check the comments under the review

------

**Optional suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 5 above)
Comment (IamYunusAli) [reply index.html]: you mean <br> ?

Comment (tufoinnkuo10) [reply index.html]: About` <br>`
Myself

like the figma design... its optional though

File: style.css (hunk 1 above)
Comment (IamYunusAli) [reply style.css:138]: changed it to flex with column

File: style.css (hunk 2 above)
Comment (IamYunusAli) [reply style.css]: changed in to the main color on figma #f7f7f9;

File: index.html (hunk 3 above)
Comment (IamYunusAli) [reply index.html:52]: the gab is 130 on figma mine is the same now

File: index.html (hunk 4 above)
Comment (IamYunusAli) [reply index.html]: used these attribute for lets connect as it given on figma
  font-weight: 500;
  font-size: 16px;
  line-height: 24px;
  color: #7f8cff;

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark: :heavy_check_mark:

Hi @IamYunusAli   

Your project is complete! Great job for making all the required changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**